components/org.wso2.carbon.siddhi.metrics.core/src/main/java/org/wso2/carbon/siddhi/metrics/core/internal/service/MetricsServiceComponent.java

```diff
@@ -47,13 +47,13 @@ public MetricsServiceComponent() {
 
     @Activate
     protected void start(BundleContext bundleContext) {
-       log.info("MetricsServiceComponent has been activated.");
+        log.debug("MetricsServiceComponent has been activated.");
```

Comment:
Can you please check if debug enabled?

Reply:
SInce we use SLF4J, no need to check `isDebugEnabled()`, see https://www.slf4j.org/faq.html#logging_performance